src/vm/appdomainnative.hpp

```diff
@@ -33,7 +33,6 @@ class AppDomainNative
     static FCDECL1(INT32, GetId, AppDomainBaseObject* refThisUNSAFE);
     static FCDECL1(INT32, GetIdForUnload, AppDomainBaseObject* refDomainUNSAFE);
     static FCDECL1(FC_BOOL_RET, IsDomainIdValid, INT32 dwId);
-    static FCDECL1(FC_BOOL_RET, IsFinalizingForUnload, AppDomainBaseObject* refThisUNSAFE);
```

Comment:
Does the [C# projection](https://github.com/dotnet/coreclr/blob/85374ceaed177f71472cc4c23c69daf7402e5048/src/System.Private.CoreLib/src/System/AppDomain.cs#L270-L273) also need to be removed? I haven't actually fully grok'd how the native/managed stubs map yet, so this might not be the right projection.